Earthfile

```diff
@@ -98,14 +98,22 @@ install-python-deps:
     FROM +install-deps
     RUN pip install wheel
     COPY demo/demo_notebooks/requirements.txt requirements.txt
+    COPY --dir python ./
+    RUN pip install --upgrade pip
     RUN pip wheel -r requirements.txt --wheel-dir=wheels
+    RUN pip wheel -r python/tests/requirements.txt --wheel-dir=wheels
+    RUN pip wheel python/ --wheel-dir=wheels
     SAVE ARTIFACT wheels /wheels
 
 install-python:
     FROM +install-deps
     COPY +install-python-deps/wheels wheels
     COPY demo/demo_notebooks/requirements.txt requirements.txt
+    COPY --dir python ./
+    RUN pip install --upgrade pip
```

Comment:
Any reason you need this? IMO this upgrades pip non-deterministically?

Reply:
I think it's related to `pyproject.toml` not being (well) supported with out-of-the-box Ubuntu 22.04 ([issue](https://github.com/pypa/setuptools/issues/3269))
An alternative would be to either switch to legacy `setup.py` temporarily or to have both (will have to how that resolves though).

Reply:
hm I thought we're already on 24.04 I swear I did the earthly upgrade at some point but it's not in main :( .. that's probably the right move for it